apps/support/Makefile.inc

```diff
@@ -103,6 +106,8 @@ CXX-host-metal ?= $(CXX)
 CXX-host-hvx_128 ?= $(CXX)
 CXX-host-hvx_64 ?= $(CXX)
 CXX-host-hvx ?= $(CXX)
+CXX-hexagon-32-qurt-hvx_128 ?= hexagon-clang++
+CXX-hexagon-32-qurt-hvx_64 ?= $(CXX-hexagon-32-qurt-hvx_128)
```

Comment:
I think we've been deprecating hvx_64, do we need this? (And same below)

Reply:
hvx_64 doesn't exist anymore. hvx_128 is just a synonym for hvx (which is now preferred). Remove hvx_64 and add hvx.